src/core/request.js

```diff
@@ -300,8 +311,7 @@ export default function request(options) {
                         code: xhr.status,
                         sent: xhr.readyState > 0,
                         type: 'error',
-                        textStatus: textStatus,
-                        message: errorThrown || __('An error occurred!')
+                        textStatus: textStatus
```

Comment:
`__` wasn't properly set up by `oat-sa/test-taker-portal` and resulted in js error. It wasn't clear for me how to fix it, so I just removed translations from here. I don't see `responseExtras.message` being used, and not translating technical `'The server has sent an empty response'` message also looks fine to me,